tests/Contracts/CEtherHarness.sol

```diff
@@ -136,7 +136,7 @@ contract CEtherDelegateHarness is CEtherDelegate {
     }
 
     function harnessSetFuseFeeFresh(uint newFuseFeeMantissa) public returns (uint) {
-        return _setFuseFeeFresh(newFuseFeeMantissa);
+        return 0; //_setFuseFeeFresh(newFuseFeeMantissa);
```

Comment:
not sure if that's related to your changes @Joeysantoro (like the error code refactor), but a comment to explain this would have been useful